report/docker_run.sh

```diff
@@ -168,5 +170,18 @@ fi
 # Wait for the report process to finish uploading.
 wait $pid_report
 
+$PYTHON -m report.trends_report \
+  --results-dir ${LOCAL_RESULTS_DIR:?} \
+  --output-path "gs://oss-fuzz-gcb-experiment-run-logs/trend-reports/${GCS_TREND_REPORT_PATH:?}" \
+  --name ${EXPERIMENT_NAME:?} \
+  --date ${DATE:?} \
+  --url "https://llm-exp.oss-fuzz.com/Result-reports/${GCS_REPORT_DIR:?}" \
+  --benchmark-set ${BENCHMARK_SET:?} \
+  --model ${MODEL:?} \
+  --tags ${FREQUENCY_LABEL:?} ${SUB_DIR:?} \
```

Comment:
Could we also record the following info in `docker_run.sh` for a fine-grained comparison?
1. `RUN_TIMEOUT`
2. `NUM_SAMPLES`
3.  `LLM_FIX_LIMIT`

This is because we speed up PR experiments with lower values for them, which will likely add unnecessary fluctuations to the trend reports.